src/region/RegionAnnotation.scss

```diff
@@ -20,3 +20,7 @@
         @include ba-RegionRect-callout;
     }
 }
+
+.ba-Layer--region {
```

Comment:
Should these various overrides all live in `BaseAnnotator.scss`?

Reply:
My concern is that the order will be hard to reason about with them spread out in separate files.

Reply:
We can! I figured since we had separate folders for region, drawing, and highlighting, that they could live there. But if `BaseAnnotator` is better, we can put it there

Reply:
Ahh I see, that's true; and most of them have the same z-index, so grouping them may be cleaner too; ok! I'll move them